apptainer/moose.def

```diff
@@ -94,6 +94,9 @@ From: {{ APPTAINER_FROM }}
     cd ..
     rm -rf moose
 
+    # Install node.js
+    dnf install -y nodejs npm
```

Comment:
@dschwen how are you obtaining `nodejs` currently (Conda, Homebrew, the system)? I am curious as to the need for another package manager on-the-fly during runtime (why `npm`?), etc.

Reply:
I usually obtain it using `nvm`. Conda is no alternative to `npm`. An the npm packages are installed in tree with specific pinned versions.